lnrpc/lightning.proto

```diff
@@ -2312,6 +2312,9 @@ message PendingChannelsResponse {
 
         // Total number of forwarding packages created in this channel.
         int64 num_forwarding_packages = 10;
+
+        // A set of flags showing the current state of the channel.
+        string chan_status_flags = 11;
```

Comment:
While string is more readable, it's less nice to consume programmatically (`strings.Contains`) vs just checking a bit in an integer. Also a change in string values could break this API. 

Definitely a tradeoff with UX here though, WDYT about surfacing the bitvector on the api then pretty-printing it in lncli?

Reply:
+1, perhaps better to have the bitvector and add a comment documenting the different chanstatuses.

Reply:
Note that this `string chan_status_flags` is already used in the message `Channel`, changing the type here will make the same field filled with different values(unless we deprecate the field there?). This field is not only used in cli but also in RPCs, in our case the `lndclient`.

Tbh I'd rather have a `chan_status` rather than a list of flags. Using a bitvector here creates cumbersome as we need to check which bit means what every time we use it vs just reading from the response to understand the channel status. And the users need to do the job that `lnd` has already done for them which is to translate the bits into interpretable strings.

Reply:
After thinking this through, I'm convinced since it's already exposed. SGTM!